user/pcsc-cyberjack/template.py

```diff
@@ -0,0 +1,29 @@
+pkgname = "pcsc-cyberjack"
+_version = "3.99.5"
+_sp = "16"
+pkgver = f"{_version}.{_sp}"
```

Comment:
pkgver should not be an fstring

Reply:
Well, the official version string is `3.99.5final.sp16`, with `3.99.5final` being released in 2011 and all the `.sp<num>` versions having been in the years since. What would you suggest in terms of versioning?.

Reply:
```
pkgver = "3.99.5.16"
_sp = pkgver[pkgver.rfind(".") + 1:]
_version = pkgver[:-len(_sp) - 1]
```

but formatted